app/controllers/articles_controller.rb

```diff
@@ -1,6 +1,10 @@
 class ArticlesController < ApplicationController
   def index
-    @articles = Article.all
```

Comment:
I'm guessing a refactoring if coming for this but you could pull all the paginations variables/logic into a class that gets initialized with the params. Then instead of checking if, for example, the params[:count].nil?, you could just set a default value for it when initialized. Just some thoughts.

Reply:
Oooooh yes that makes sense! Thanks for the suggestion 😄 
Also good guess 😂 , I was going to do that in Part 3 of this Pagination section!